Gemfile

```diff
@@ -1,46 +1,5 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
-
-branch = ENV.fetch('SOLIDUS_BRANCH', 'master')
-solidus_git, solidus_frontend_git = if (branch == 'master') || (branch >= 'v3.2')
-                                      %w[solidusio/solidus solidusio/solidus_frontend]
-                                    else
-                                      %w[solidusio/solidus] * 2
-                                    end
-gem 'solidus', github: solidus_git, branch: branch
```

Comment:
The different versions are handled at the dummy-app level, we could keep that, but since we're not running much within the extension anymore it's no longer required.

Reply:
Not sure I follow. We still have a bunch of models we probably want to ensure they're working with different Solidus versions. How will we run CI on older versions if we remove the expected logic around env vars? 🤔

Reply:
Although we don't have the CI harness to leverage that yet, we still have the env var support in `bin/dummy-app` here https://github.com/solidusio/solidus_paypal_commerce_platform/blob/f9b9b31ed686af75007f8a5e425c5429c0c1007a/bin/dummy-app#L32

I think we can use that without the need to also handle it in the main Gemfile, wdyt?

Reply:
I see. Probably the decision here is coupled to https://github.com/solidusio/solidus_paypal_commerce_platform/pull/166#discussion_r1032533834

Reply:
Yes indeed, the current setup doesn't run specs through this Gemfile anymore 👍